syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message ResultCode {
    uint32 errcode = 1;
    bytes errmsg = 2; 
}

message GetFriendListRequest {
    uint32 id = 1;
    bytes name = 2;
    bytes pwd = 3;
}

message GetFriendListRespone {
    ResultCode result = 1;
    repeated bytes friendList = 2;
    bool success = 3;
}

service FriendServiceRpc {
    rpc GetFriendList(GetFriendListRequest) returns(GetFriendListRespone);
}


